css/styles.css

```diff
@@ -0,0 +1,839 @@
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+  margin: 10px auto;
+  padding: 12px;
+  margin-bottom: 5px;
+}
+
+.about p {
+  max-width: 920px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 400;
+  font-size: 20px;
+  line-height: 32px;
+  color: #42526e;
+  text-align: center;
+  margin: 10px 5px;
+}
+
+.about .p-wrapper {
+  position: relative;
+}
+
+.about .p-wrapper::before {
+  content: "";
+  display: block;
+  position: absolute;
+  left: -60px;
+  bottom: -65px;
+  height: 56px;
+  width: 126px;
+  background-image: url(../images/about-left.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.about .p-wrapper::after {
+  content: "";
+  display: block;
+  position: absolute;
+  right: -55px;
+  top: -130px;
+  height: 127px;
+  width: 100px;
+  background-image: url(../images/about-right.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+@media (min-width: 768px) {
+  .about .p-wrapper::before {
+    left: -24px;
+    width: 231px;
+  }
+
+  .about .p-wrapper::after {
+    width: 145px;
+  }
+}
+
+.btn {
+  width: 159px;
+  padding: 12px;
+  margin-bottom: 0;
+  display: inline-block;
+}
+
+.cardme {
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  max-width: 1171px;
+  width: 100%;
+  gap: 24px;
+  align-items: center;
+}
+
+.cards-work {
+  display: flex;
+  flex-direction: column;
+  background-image: url(/imagestwo/ImgPlaholderDesk.svg);
+  background-color: #ebf0ee;
+  border-radius: 8px;
+  max-width: 367px;
+  width: 100%;
+  height: 355px;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+}
+
+.figcard figure {
+  width: 80px;
+  height: 80px;
+  border-radius: 50%;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 40px;
+}
+
+.languaged {
+  max-width: 337px;
+  height: 96px;
+  gap: 24px;
+  margin: auto;
+  border: 4px;
+}
+
+.title {
+  text-align: center;
+  height: 24px;
+  font-weight: 600;
+  margin-bottom: 15px;
+  font-size: 24px;
+  font-family: 'Inter', sans-serif;
+}
+
+.uld-tags {
+  /* width: 89px; */
+  display: flex;
+  gap: 8px;
+
+  /* margin-top: 16px; */
+  width: 271px;
+  height: 40px;
+  justify-content: center;
+  align-items: center;
+  padding-left: 2px;
+  margin: auto;
+}
+
+.btn-tag {
+  background: #fff;
+  border-radius: 4px;
+  border: 0;
+  width: 89;
+  height: 28px;
+  padding: 10px;
+}
+
+.li-tag-list {
+  list-style: none;
+  height: 30px;
+  color: #36b37e;
+  font-weight: 600;
+  font-size: 12px;
+  letter-spacing: 0.03em;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.form {
+  /* border: 2px solid skyblue; */
+  max-width: 1171px;
+  margin: auto;
+  display: flex;
+  padding: 80px 0;
+  gap: 24px;
+}
+
+@media (min-width: 768px) {
+  .form {
+    display: flex;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] Please also fix the horizontal paddings and margins of the `form` section, as it is sticking to the page borders, and it also should match the design of the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468).

Right now it's like this:
![image](https://user-images.githubusercontent.com/20567503/178080905-9140312d-81ae-4b4f-b1a7-52a73e4529f0.png)

This is the design of the Figma template:

![image](https://user-images.githubusercontent.com/20567503/178080950-9c30d624-3a87-4a69-8a92-affeaa6dbc5e.png)

Reply:
Resolved